Sources/MapboxCoreNavigation/Logger+Internal.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+import OSLog
+@_implementationOnly import MapboxCommon_Private.MBXLog_Internal
+
+/// :nodoc:
+public struct Log {
```

Comment:
Do we need to make this public? I'm afraid `Log` is too common for a struct name

Reply:
SPI sounds reasonable. Also, as an alternative, we can make a verbose name (i.e. `NavigationLog`) and leave this public, but have additional `internal typalias Log = NavigationLog`.